crates/bws/CHANGELOG.md

```diff
@@ -7,13 +7,17 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.0.0] - 2024-09-26
+
 ### Added
 
 - The ability to edit unassigned secrets with direct permissions. (#906)
+- Add a way to opt out of state by default with the new state_opt_out key (#930)
 
```

Comment:
```suggestion
- `run` command to run commands with secrets (#621)

```

Reply:
Thanks, I missed the run command. Added in 5411f735fa00280d61eb1ee2049847789670e810